pyomo/gdp/plugins/bigm_mixin.py

```diff
@@ -23,7 +27,7 @@ def _convert_M_to_tuple(M, constraint, disjunct=None):
         else:
             try:
                 M = (-M, M)
-            except:
+            except Exception:
```

Comment:
You touched this line so now I get to snark. It's good practice, if catching a general exception, to raise the original exception as well so folks can directly inspect it.